client/__tests__/e2e/cellxgeneActions.js

```diff
@@ -312,4 +315,75 @@ export async function assertCategoryDoesNotExist(categoryName) {
 
   await expect(result).toBe(false);
 }
+
+export async function login() {
+  const email = `cellxgene-smoke-test+${process.env.DEPLOYMENT_STAGE}@chanzuckerberg.com`;
+  const password = "Test1111";
```

Comment:
We just have to make sure the permissions for these test accounts are barebones since anyone who looks at this file will have access to the credentials

Reply:
we only have one permissions state and that is logged in or not. We can create more in future. This test doesn't run on prod does it?

Reply:
oh the test is intended to run on prod too! But I haven't created a prod Auth0 account yet, tentatively `cellxgene-smoke-test+prod@chanzuckerberg.com`, because I wanna wait till prod deploy and see if https://cellxgene.cziscience.com/experimental/krasnow_lab_human_lung_cell_atlas_10x-1.cxg becomes available!

Reply:
Since the infra PR is not merged yet, the integration test won't run for all envs at the moment: https://github.com/chanzuckerberg/single-cell-infra/pull/198/files